app/models/author.rb

```diff
@@ -5,4 +5,6 @@ class Author < ApplicationRecord
   validates :name, presence: true, length: {maximum: Settings.length.digit_50}
   validates :dob, presence: true
   validates :info, presence: true
+
+  scope :order_name, ->{order :name}
```

Comment:
```suggestion
  scope :order_by_name, ->{order :name}
```
default nó order asc hay desc vậy e ?

Reply:
default là asc ạ :v nên e mới ko config